packages/mobile/src/screens/profile-screen/ProfileHeader/ProfileInfoTiles.tsx

```diff
@@ -67,29 +76,63 @@ const useInfoTileStyles = makeStyles(({ spacing }) => ({
 
 const messages = {
   aiGeneratedTracks: 'AI Generated Tracks',
+  comments: 'Comments',
   mutuals: 'Mutuals',
   relatedArtists: 'Related Artists',
   viewAll: 'View All',
   tipSupporters: 'Tip Supporters',
+  topSupporters: 'Top Supporters',
   supportedUsers: 'Supporting'
 }
 
 type ProfileInfoTileProps = {
-  style?: ViewStyle
   screen: string
   icon: ComponentType<SvgProps>
   title: string
   content: React.ReactNode
+  showCount?: boolean
+}
+
+const ProfileInfoTile = (props: ProfileInfoTileProps) => {
+  const { screen, icon: Icon, title, content } = props
+  const styles = useInfoTileStyles()
+  const { neutral } = useThemePalette()
+  const navigation = useNavigation()
+  const { user_id } = useSelectProfile(['user_id'])
+
+  const handlePress = useCallback(() => {
+    navigation.navigate(screen, { userId: user_id })
+  }, [navigation, screen, user_id])
+
+  return (
+    <Paper
+      h={64}
+      style={styles.tileContent}
+      border='default'
+      shadow='near'
+      onPress={handlePress}
+    >
+      <View style={styles.title}>
+        <Icon height={20} width={20} fill={neutral} />
```

Comment:
color="default" also works. Also double check icon height, and maybe just use size prop?